java/app/src/main/java/com/fullstorydev/shoppedemo/data/ProductRepository.java

```diff
@@ -1,30 +1,50 @@
 package com.fullstorydev.shoppedemo.data;
 
 import android.app.Application;
+import android.os.AsyncTask;
 
 import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
 
+import com.fullstorydev.shoppedemo.R;
+import com.fullstorydev.shoppedemo.utilities.JsonHelper;
+import com.fullstorydev.shoppedemo.utilities.NetworkUtils;
+import com.google.gson.JsonParseException;
+
+import java.io.IOException;
 import java.util.List;
 
 public class ProductRepository {
 
     private ProductDao mProductDao;
-    private LiveData<List<Product>> mAllProducts;
+    private LiveData<List<Product>> mAllItems; //items are in cart
+    private MutableLiveData<List<Product>> mAllProducts; //products are sold in shop, fetched from API
```

Comment:
Just curious: why use `LiveData` for cart items but `MutableLiveData` for products?